packages/commons-ui-core/README.md

```diff
@@ -0,0 +1,17 @@
+# @commons-ui/core
+
+### Getting started
+
+To run commons-ui core test:
+
+```
+pnpm test
+
+```
```

Comment:
This is not running

Reply:
@kilemensi Could reinstall the dependencies again? Mine is running 👇🏽
<img width="1115" alt="Screenshot 2022-03-29 at 13 04 58" src="https://user-images.githubusercontent.com/3693563/160587685-99e41e5f-704c-4f8f-b092-82b6ce859693.png">